sonaranalyzer-dotnet/rspec/vbnet/S2077_vb.net.html

```diff
@@ -68,19 +68,10 @@ <h2>Sensitive Code Example</h2>
     context.Query(Of User)().FromSql(sensitiveQuery) ' Sensitive
 End Sub
 </pre>
-<h2>Exceptions</h2>
-<p>No issue will be raised in the following cases:</p>
-<ul>
-  <li> When the SQL query is a FormattableString provided <strong>directly</strong> to <code>ExecuteSqlCommand</code>,
-  <code>ExecuteSqlCommandAsync</code> or <code>FromSql</code>. In this case the FormattableString is automatically converted to a parametrized query.
-  This feature was added in Entity Framework Core 2.0. See <a
-  href="https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-2.0#string-interpolation-in-fromsql-and-executesqlcommand">Entity Framework's
-  documentation</a> for more information. </li>
-</ul>
+<h2>Compliant Solution</h2>
 <pre>
 Public Sub Foo(ByVal context As DbContext, ByVal value As String)
-    context.Database.ExecuteSqlCommand("SELECT * FROM mytable") ' No issue raised. The query is hard-coded. Thus no injection is possible.
-    context.Database.ExecuteSqlCommand($"SELECT * FROM mytable WHERE mycol={value}") ' No issue raised. The FormattableString is transformed into a parametrized query.
+    context.Database.ExecuteSqlCommand($"SELECT * FROM mytable WHERE mycol=@p0", param) ' Compliant, the query is parametrized
```

Comment:
is the interpolated string necessary here? it's using `@p0` rather than `{value}` as before

Reply:
Indeed there are multiple problems here:
- $ is not needed
- the parameter is called **value** and not **param**
- typo at **parametrized**

I've updated the docs and regenerated the rpsec.